frontend/src/components/config-details.ts

```diff
@@ -295,7 +295,19 @@ export class ConfigDetails extends LiteElement {
         msg("List of URLs"),
         html`
           <ul>
-            ${this.seeds?.map((seed: Seed) => html` <li>${seed.url}</li> `)}
+            ${this.seeds?.map(
+              (seed: Seed) =>
+                html`
+                  <li>
+                    <a
+                      class="text-primary hover:text-indigo-400"
+                      href="${seed.url}"
+                      target="_blank"
```

Comment:
Let's add [rel="noreferrer"](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer) when linking to external sites for extra privacy